docs/source/api/v2/servers_details.rst

```diff
@@ -0,0 +1,117 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-servers-details:
+
+*******************
+``servers/details``
+*******************
+Retrieves details of :ref:`tp-configure-servers`.
+
+
+``GET``
+=======
+:Auth. Required: Yes
+:Roles Required: "read-only"
+:Response Type:  Array
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+----------------+----------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Name           | Required                               | Description                                                                                                                                                    |
+	+================+========================================+================================================================================================================================================================+
+	| hostName       | Required if no physLocationID provided | Return only the servers with this (short) hostname. Capitalization of "hostName" is important.                                                                 |
+	+----------------+----------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| physLocationID | Required if no hostName provided       | Return only servers with this integral, unique identifier for the physical location where the server resides. Capitalization of "physLocationID" is important. |
```

Comment:
How would you capitalize an id?

Reply:
I might be inclined to capitalize it `physLocationId`, which is how GET `deliveryservices/{{ID}}/servers/eligible`, GET `deliveryservices/{{ID}}/servers`, GET `deliveryservices/{{ID}}/unassigned_servers`, GET `servers/{{ID}}`, and GET `servers` all capitalize it. This is the only route in Traffic Ops that capitalizes it this way.